extension/wallet/Cargo.toml

```diff
@@ -15,6 +15,8 @@ aes-gcm-siv = { version = "0.9", features = [ "std" ] }
 anyhow = "1"
 baru = "0.1"
 bdk = { version = "0.4", default-features = false }
+bip32 = { version = "0.2", features = [ "secp256k1" ] }
+bip39 = { version = "1", features = [ "rand" ] }
```

Comment:
The `bip32` crate comes with bip39 support, why an extra dependency?

Reply:
Addressed in follow-up PR here: https://github.com/comit-network/waves/pull/193